Packages/Tracking/Core/Runtime/Plugins/LeapCSharp/Connection.cs

```diff
@@ -873,23 +858,13 @@ public void SetAndClearPolicy(Controller.PolicyFlag set, Controller.PolicyFlag c
             UInt64 clearFlags = (ulong)FlagForPolicy(clear);
             eLeapRS result;
 
-            if (device == null || !_multiDeviceAwareConnection)
+            if(device != null && Controller.CheckRequiredServiceVersion(MinServiceVersionForMultiModeSupport, this))
             {
-                result = LeapC.SetPolicyFlags(_leapConnection, setFlags, clearFlags);
+                result = LeapC.SetPolicyFlagsEx(_leapConnection, device.Handle, setFlags, clearFlags);
             }
             else
             {
-                if (!Controller.CheckRequiredServiceVersion(MinServiceVersionForMultiModeSupport, this))
-                {
-                    UnityEngine.Debug.LogWarning(String.Format("Your current tracking service does not support setting policy flags on a per device basis (min version is {0}.{1}.{2}). Please update your service: https://developer.leapmotion.com/tracking-software-download",
-                        MinServiceVersionForMultiModeSupport.major,
-                        MinServiceVersionForMultiModeSupport.minor,
-                        MinServiceVersionForMultiModeSupport.patch));
-
-                    return;
```

Comment:
do we need this check still?

Reply:
The version check exists, there is just no warning about versions - essentially, any connection past that release will do specific-device API calls now